django/contrib/postgres/aggregates/general.py

```diff
@@ -1,14 +1,63 @@
 from django.contrib.postgres.fields import JSONField
 from django.db.models.aggregates import Aggregate
+from django.db.models.expressions import F, OrderBy
 
 __all__ = [
     'ArrayAgg', 'BitAnd', 'BitOr', 'BoolAnd', 'BoolOr', 'JSONBAgg', 'StringAgg',
 ]
 
 
-class ArrayAgg(Aggregate):
+class OrderableAgg(Aggregate):
+    template = "%(function)s(%(expressions)s %(ordering)s)"
+
+    def __init__(self, expression, ordering=None, **extra):
+        if not isinstance(ordering, list) and ordering is not None:
```

Comment:
`isinstance(ordering, (list, tuple))` (and add a test for the tuple case)
You can also borrow some documentation from ref/models/options.html (Options.ordering) -- the current proposed docs don't describe the string case or the fact that ordering can be a list/tuple.